kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/ActionsIterator.java

```diff
@@ -60,7 +60,14 @@ public class ActionsIterator implements CloseableIterator<ActionWrapper> {
    */
   private final LinkedList<DeltaLogFile> filesList;
 
-  private final StructType readSchema;
+  /** Schema used for reading delta files. */
```

Comment:
nit: update the class docs to indicate two schemas of data.

Reply:
Done.